src/resources/Change/SP.ts

```diff
@@ -7,6 +7,7 @@ export const SpChangeResource: ResourceIF = {
   contactLabel: 'Business',
   displayName: GetCorpFullDescription(CorpTypeCd.SOLE_PROP),
   addressLabel: 'Business Addresses',
+  showEmail: true,
```

Comment:
In the future, when you add a new resource like this, you might set the default value to get existing behaviour, so that you only have to impact the filings you are working on. Here, you have impacted (though low risk) some unrelated filings.

On the other hand, I do like resources/props to ENABLE a feature, as you have done here.

Also, you have already implemented this, so no change required. (Just be aware for next time, to reduce risk and testing impact.)

Reply:
Although this is low impact, I will change the implementation based on your suggestion (^^ comment). I think it is unnecessary to change unrelated files and can be handled with default value and prop..

Reply:
Sounds good.